akka-actor-typed-tests/src/test/java/jdocs/akka/typed/InteractionPatternsTest.java

```diff
@@ -314,6 +355,122 @@ public void timers() throws Exception {
     Await.ready(system.terminate(), Duration.create(3, TimeUnit.SECONDS));
   }
 
+
+
+  // #actor-ask
+  interface HalProtocol {}
```

Comment:
`HalCommand` or `HalRequest` ?

Reply:
Good idea, I'll try to follow that pattern for the other ones as well, makes more sense.